### ISSUE TITLE
JAP Hirohito coups japan and resets it to the 1962 focus tree (with the oil crisis tree)

### ISSUE THREAD

**Issue by jonteboi0707** [TNODevs/TNO-Bug-Tracker#3420]
### Operating System

Windows 11

### HOI4 version

11.14.7

### TNO version

1.5.0d

### Do you have other mods in use?

no

### Were you using Steam?

Yes

### Which major expansions do you NOT have?

I have all

### Describe the bug

Kaya's goverment collapesd and got replaced with hirohito and reset it to the starting focus tree (with the oil crisis focus tree).

### Steps to reproduce / Cause of the issue

1 Kaya gets in charge (in the game he did the hardliner path.)
2 The goverment collapsed (in game it collapesd during the oil crisis.)
3 The conservatives take over (with hirohito for some reasone???)

### Expected behaviour

I think that the goverment collapsed and switched to the conservaties who was under hirohito for some reason.e

### Screenshots

![20240720181555_1](https://github.com/user-attachments/assets/94ab2247-bbee-4d08-b23a-fbf582edbc81)
![20240720181547_1](https://github.com/user-attachments/assets/fed96733-e63c-4d64-8e72-7ccc5d73f8e1)

**Comment by Jack121Q** [comment]
Yeah, I can tell what the issue is. However, the Second Interregnum is already not supported. Consider it a failstate.

**Comment by Marok132** [comment]
> Yeah, I can tell what the issue is. However, the Second Interregnum is already not supported. Consider it a failstate.

What is the Second Interregnum and why is it not supported anymore?

**Comment by Jack121Q** [comment]
Second Interregnum happens when a Japanese PM, after Ino, gets ousted. It just rolls the mini-game to pick a leader again if the player wishes, or alternatively fires the close game screen. Until a leader is selected, Hirohito is in power (As far as I remember).
For obvious reasons, playing two PMs one after another can cause issues, I opted to keep the Second Interregnum in since some may want to try it but it breaks the narrative of Japan and is not balanced.
